themes/bootstrap3/templates/_ui/components/confirm-menu.phtml

```diff
@@ -0,0 +1,42 @@
+<?php
+  $toggleLabel = $this->transEsc($this->label ?? '{->label}');
+  $toggleClass = trim('confirm__toggle ' . $this->escapeHtmlAttr(($this->toggleClass ?? 'btn')));
+
+  $desc = $this->transEsc($this->description ?? 'Are you sure?');
+
+  $actionType = $this->action->type ?? 'submit';
+
+  $confirmLabel = $this->transEsc($this->confirm ?? 'confirm_dialog_yes');
+  $cancelLabel = $this->transEsc($this->cancel ?? 'confirm_dialog_no');
+?>
+
+<div class="confirm-menu">
+  <button class="<?=$toggleClass ?>"><?=$toggleLabel ?></button>
+
+  <div class="confirm__options">
+    <p class="confirm__desc"><?=$desc ?></p>
```

Comment:
It might be a good idea to use a naming convention where escaped variables have "escaped" in their name -- e.g. `$descEscaped` instead of `$desc`. Otherwise, it's harder to review, since it looks at first glance like escaping may have been forgotten in several places, even though it really hasn't.